ecocode-rules-specifications/src/main/rules/EC4/php/EC4.asciidoc

```diff
@@ -1,29 +0,0 @@
-Prefer local variables as parameters
```

Comment:
I don't agree to delete PHP EC4 because not deprecated and still in usage (check "EC4" rule in array above)

Reply:
ok, reverted